aten/src/TH/THAllocator.c

```diff
@@ -143,28 +193,39 @@ static void *_map_alloc(void* ctx_, ptrdiff_t size)
   if (ctx->flags & TH_ALLOCATOR_MAPPED_SHAREDMEM)
   {
     char *filename;
+    char *eventname;
     LARGE_INTEGER hfilesz;
 
     if (ctx->filename[0] == '/')
       filename = ctx->filename + 1;
     else
       filename = ctx->filename;
 
+    if (ctx->eventname[0] == '/')
+      eventname = ctx->eventname + 1;
+    else
+      eventname = ctx->eventname;
```

Comment:
Can't we put merge this `if` with the previous one? `filename` is guaranteed to be a prefix of `eventname`

Reply:
Sure.